sdk/openai/Azure.AI.OpenAI/src/Generated/Internal/ClientUriBuilder.cs

```diff
@@ -105,11 +105,7 @@ public void AppendQuery(string name, string value, bool escape)
             Argument.AssertNotNullOrWhiteSpace(name, nameof(name));
             Argument.AssertNotNullOrWhiteSpace(value, nameof(value));
 
-            if (QueryBuilder.Length == 0)
```

Comment:
Changes to the `/Generated/` folder won't persist across refreshes -- we'd need the change to happen in `autorest.csharp` for it to stay "sticky," and that'd need to be the right fix for all clients.

I'm suspicious that we're using it wrong. Where does the extra `?` character originate from?

Reply:
This is a difference between how System.UriBuilder works in .Net Framework 4.6.2, vs how it works in .Net Core 3.1, .Net 5.0 and newer. .Net Framework 4.6.2 is where we get the double ?. If we are planning to target .Net Framework, we should address this. For now I will experiment with suppressing the generation of this file and manually implementing it with correct behaviour, or just supressing the double ? at a higher level